ams/CleanupCommand.cs

```diff
@@ -28,7 +28,16 @@ internal class CleanupCommand : BaseMigrator
 
         public override async Task MigrateAsync(CancellationToken cancellationToken)
         {
-            var account = await GetMediaAccountAsync(_options.AccountName, cancellationToken);
+            MediaServicesAccountResource? account = null;
+            try
+            {
+               account = await GetMediaAccountAsync(_options.AccountName, cancellationToken);
+            }
+            catch (Exception)
+            {
+                _logger.LogError("No valid media account was found."); 
+                return;
+            }
```

Comment:
This change can be removed as well.

Reply:
This design was discussed with @pohhsu because, after obtaining the storage account, it might be confusing for the user. There's a chance they could mistakenly input "storage account" instead of the AMS (Azure Media Services) account for reset/cleanup and other commands. In such cases, this is a place handle the manually input error. i return it here but as per @duggaraju it's better to throw